cluster/saltbase/salt/etcd/etcd.manifest

```diff
@@ -1,3 +1,19 @@
+{% set cluster_state = 'new' -%}
+{% set host_name = pillar.get('master_name', '') -%}
+{% set etcd_cluster_array = pillar.get('initial_etcd_cluster', '').split(',') -%}
+{% set etcd_cluster = '' -%}
+{% set vars = {'etcd_cluster': '', 'cluster_state': cluster_state} -%}
+{% for host in etcd_cluster_array -%}
+  {% if etcd_cluster != '' -%}
```

Comment:
I think it'll be easier if etcd_cluster is an array and you use `join` function at the end.